test/e2e/manifest/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash -x
```

Comment:
Is this file used somewhere?

Reply:
This is just a helper script to deploy everything for operator testing. Maybe I should remove it